src/components/transactions/transactionsList.js

```diff
@@ -39,7 +39,7 @@ class TransactionsList extends React.Component {
     const filteredTransactions = transactions.filter(fixIncomingFilter);
 
     const isLoading = loading.filter(type =>
-      actionTypes.transactionsLoaded === type).length > 0;
+      actionTypes.getTransactionsSuccess === type).length > 0;
```

Comment:
Should be the same action as in https://github.com/LiskHQ/lisk-hub/blob/da0946a4f0b54281e404d732cc2fb501c6bf5904/src/actions/transactions.js#L65
```suggestion
      actionTypes.getTransactions === type).length > 0;
```

because otherwise the loading bar inside the transactions box on wallet page is not displayed when loading.